index.html

```diff
@@ -64,13 +64,29 @@ <h1 class="page-title">Contrast Ratio Calculator</h1>
         </div>
       </section>
 
-      <!--TODO: replace lorem ipsum with actual content-->
       <section class="secondary">
         <div class="container">
-          <h2>H2 Notes or "How-To-Use" section</h2>
-          <p>Custom color variables were added under the root element. I added section tags for the primary and secondary sections and moved the container div tag inside of each. I added a slight background color to the primary section. I also reduced the border of the pass/fail circles from 4px to 3px. And I gave the footer a slightly lighter color. Finally, I added styling for the input buttons and sized the swatch to match the input height.</p>
-          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Incidunt optio assumenda animi sint. Illo veritatis quos vitae quam provident accusamus. Atque dolor minus asperiores hic consequatur deserunt odit accusamus nisi repudiandae illum natus harum, temporibus mollitia eligendi pariatur molestiae dicta excepturi quo maxime! Mollitia excepturi libero autem molestiae quisquam adipisci.</p>
-          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Incidunt optio assumenda animi sint. Illo veritatis quos vitae quam provident accusamus. Atque dolor minus asperiores hic consequatur deserunt odit accusamus nisi repudiandae illum natus harum, temporibus mollitia eligendi pariatur molestiae dicta excepturi quo maxime! Mollitia excepturi libero autem molestiae quisquam adipisci.</p>
+          <h2>How does it work?</h2>
+          <p>
+            Enter a foreground and background color in RGB hexadecimal format (e.g., #000 or #FF00FF).
```

Comment:
I think it's good for the start, but we decided that user will have opprotunity to enter colors in other formats also I guess. 
e.g.: rgb(220, 112, 57)
I will not request changes here, since reviews requested from other guys, and this changes can be applied later easily.

Reply:
I add the line "How to use" just as a filler. I think the point about other formats is valid, but mentioning only RGB is fine since we don't have anything working yet. I assume we can add text as features come online. 

The notes on AA vs AAA and the reference numbers are good - I didn't know that. So "How to use" might be about "How to fix" if the user's numbers fail the tests. And feel free to change the paragraphs to unordered lists or other HTML elements. I have no idea how it's done in the real world. Maybe you could make an alternate section or two, but I don't want to overwhelm you.